crates/store/re_grpc_client/src/connection_registry.rs

```diff
@@ -48,6 +50,38 @@ impl ConnectionRegistry {
     }
 }
 
+/// Possible errors when creating a connection.
+#[derive(Debug, thiserror::Error)]
+pub enum ClientConnectionError {
+    /// Native connection error
+    #[cfg(not(target_arch = "wasm32"))]
+    #[error("Connection error: {0}")]
+    Tonic(#[from] tonic::transport::Error),
```

Comment:
Have you checked the formatting of this one?

Reply:
No, but it appears to be not too crazy.

<img width="690" height="178" alt="Image" src="https://github.com/user-attachments/assets/aa1aa8b9-4d64-4699-812b-03b9da901be5" />